### PR TITLE
Initial project setup

### PR DESCRIPTION
**Pull request by burzum619**
https://jira.fairwaytech.com/browse/TDS-751 

Basic project setup/some basic dependencies that will be used by the service.

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -0,0 +1,187 @@
...
+			<artifactId>mockito-all</artifactId>
+			<version>${mockito.version}</version>
+			<scope>test</scope>
+		</dependency>
+		<dependency>
+			<groupId>io.github.benas</groupId>
+			<artifactId>random-beans</artifactId>
+			<version>3.5.0</version>
+			<scope>test</scope>
+		</dependency>
```
Comment (wtritch) [inline pom.xml:110]: Make sure you have spring-security for actuator endpoint security

File: pom.xml [hunk 2]
```diff
@@ -0,0 +1,187 @@
...
+
+	</dependencies>
+
+	<build>
+		<plugins>
+			<plugin>
+				<groupId>org.springframework.boot</groupId>
+				<artifactId>spring-boot-maven-plugin</artifactId>
+			</plugin>
```
Comment (wtritch) [inline pom.xml:119]: Might need the maven-release-plugin in here

**Review by wtritch — APPROVED** [review]
LGTM

File: pom.xml (hunk 1 above)
Comment (burzum619) [reply pom.xml:110]: Does that need to be included as a separate dependency? I don't see any spring-security dependencies in our other microservices

Comment (millerts) [reply pom.xml:110]: I think we may get that by puling in common.  I'll let @wtritch confirm

**Review by millerts — APPROVED** [review]

File: pom.xml (hunk 1 above)
Comment (wtritch) [reply pom.xml:110]: ah, you're absolutely right.  It's coming from TDS_Common

Comment (millerts) [reply pom.xml:110]: So I think if we add a dependency to common we'll be good to go.   And also add it to the base configuration.  We should have a config class like [this](https://github.com/SmarterApp/TDS_ExamService/blob/develop/service/src/main/java/tds/exam/configuration/web/ExamServiceApplicationConfiguration.java) minus the cache and exception advice configs

**Pull request merged**